packages/image_picker/image_picker_platform_interface/lib/src/types/media_selection_options.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:image_picker_platform_interface/image_picker_platform_interface.dart';
+
+/// Options class for [ImagePickerPlatform.getMedia].
+class MediaSelectionOptions {
+  /// Construct a new MediaSelectionOptions instance.
+  MediaSelectionOptions({
+    List<RetrieveType>? types,
+    this.maxImageWidth,
+    this.maxImageHeight,
+    this.imageQuality,
```

Comment:
We keep repeating this triplet of arguments; we should extract an `ImageAdjustmentOptions` (or similar; that's not a great name but I don't have a better suggestion offhand).

Reply:
Maybe `ImageSelectionOptions`.

Reply:
Agreed, moved it into `ImageSelectionOptions`.